### ISSUE TITLE
First-come, First-serve process scheduling

### ISSUE THREAD

**Issue by Ram-Aditya** [WebClub-NITK/Hacktoberfest-2k18#16]
### Description
- Implement the First-Come, First-Serve process scheduling algorithm. 

- The input to your algorithm must be a 2-D array that contains values for(Burst Time, Arrival Time) for 10 processes. Initialize random values for each of the 2 attributes for each process and print this table. 
- The expected output is the list of statements that show the order in which processes were scheduled and at what time they were scheduled.
-  Acceptable range of values for input:
  Burst Time: [0,10]microseconds
  Arrival Time: [0,50]seconds
  *Burst Time is the amount of time the process requies to execute on the CPU.

### Details
- Technical Specifications:
  + You must use any one of the following programming languages: Python, Java, C, C++.
- Type of issue: No more than one implementation will be accepted for each language. Once the implementation for a language has been taken up (i.e if it is checked in the 'Issue Progress' section), you must choose from the remaining. 
- Time Limit: Issues will be given on a first-come, first-serve basis. Once you take this issue up, you must complete it within 2 days, failing which the issue will be allocated to the next person who requested for this issue.

- Additional Instructions:
   + While making the Pull Request, attach screenshots of 5 sample inputs and their corresponding output.

### Issue progress
- [ ] Python
- [ ] C++
- [ ] Java
- [ ] C

### Resources
https://www.tutorialspoint.com/operating_system/os_process_scheduling_algorithms.htm

### Directory Structure
 For this issue, use the following directory of theHacktoberfest-2k18 repository:
`/systems/os/process_scheduling/fcfs/language_name`
(where language_name is one of the languages mentioned above.)

**Comment by insane-abhi** [comment]
I want to take upon this issue.

**Comment by Ram-Aditya** [comment]
Which language would you like to take it up in?

**Comment by Ram-Aditya** [comment]
@insane-abhi  Which language would you like to take it up in?

**Comment by insane-abhi** [comment]
C++

**Comment by Ram-Aditya** [comment]
@insane-abhi Yes sure, you can go ahead. But please make sure to submit within 2 days

**Comment by AmeyCaps** [comment]
I am working on it's implementation in C.

**Comment by Ram-Aditya** [comment]
@AmeyCaps  Sure, you can go ahead. But please make sure to make the pull request within 2 days.

**Comment by Ram-Aditya** [comment]
@AmeyCaps Please try to submit by tomorrow, else the implementation in C will have to be handed over to the next person requesting it.

**Comment by Ram-Aditya** [comment]
@AmeyCaps  Sorry, but the implementation in C will now have to be handed over to the next person requesting it. Although you can still claim the issue for implementation in Java or Python.

**Comment by pramathabhat** [comment]
I would like to work on this issue.

**Comment by syk007** [comment]
I claim to implement in python

**Comment by Ram-Aditya** [comment]
Yes sure you can go ahead. But please make sure to submit within the time
limit.

On 13 Oct 2018 19:57, "syk007" <notifications@github.com> wrote:

I claim to implement in python

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/WebClub-NITK/Hacktoberfest-2k18/issues/16#issuecomment-429546546>,
or mute the thread

**Comment by syk007** [comment]
I have done it kindly check it @Ram-Aditya
